hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -35,9 +35,9 @@
     private String fileId;
 
     /**
-     * Full path to the file on underlying file system
+     * Relative path to the file from the base path
      */
-    private String fullPath;
+    private String path;
```

Comment:
Since we are renaming this, I 'd just like to suggest 1 quick test of reading a old commit metadata that contains the filed `fullPath` using this new code and making sure nothing crashes.. It should n't given `ignoreUnknown = true` , but something to test out. cc @prazanna